tests/unit/mesh/test_ncmesh.cpp

```diff
@@ -120,7 +120,8 @@ TEST_CASE("NCMesh 3D Refined Volume", "[NCMesh]")
    auto mesh_fname = GENERATE("../../data/ref-tetrahedron.mesh",
                               "../../data/ref-cube.mesh",
                               "../../data/ref-prism.mesh",
-                              "../../data/ref-pyramid.mesh"
+                              "../../data/ref-pyramid.mesh",
+                              "data/tinyzoo-3d.mesh"
```

Comment:
Also include `../../data/tinyzoo-3d.mesh`? Why are there 2 versions of this mesh?